### ISSUE TITLE
Accessibility updates to color contrast 

### ISSUE THREAD

**Issue by lwrubel** [gwu-libraries/sfm-ui#1022]
Color contrast insufficient:

- Blue text overall is not enough contrast on white or on gray
-- Bright blue (#007BFF) against white is 3.97 not above 4.5:1
- Navigation bar link contrast:
--  #888888 text on #F8F9FA is currently 3.36:1
- Dropdown buttons
--Background color is same as text when selecting options in the Turn Off and Deactivate buttons and user profile dropdown. 

Update colors of collection detail page buttons (Edit, Turn off, Deactivate) as part of this. 

Use as a basis https://creativeservices.gwu.edu/color-palette

For further accessibility details see #1004
